modules/swagger-jaxrs/pom.xml

```diff
@@ -103,13 +103,8 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>com.fasterxml.jackson.dataformat</groupId>
-            <artifactId>jackson-dataformat-yaml</artifactId>
```

Comment:
swagger-core module still has this dependency so this will be inherited transitively by swagger-jaxrs. If you prefer, I can add back the depency explicitly here too. It's more useful to remove the other 2 dependencies (jackson-jaxrs-json-provider and jackson-dataformat-xml)